psychopy/app/builder/builder.py

```diff
@@ -2111,7 +2111,7 @@ def renameRoutine(self, name, event=None, returnName=True):
             # namespace:
             name = exp.namespace.makeValid(
                 name, prefix='routine')
-            if oldName in self.exp.routines.keys():
+            if oldName in list(self.exp.routines.keys()):
```

Comment:
In Python 2, `d.keys()` returns a list. Python 3 returns a `dict_keys` view. Therefore, if the old behavior was really intended, it is correct to explicitly convert the return value into a list. However, in this particular case I would simply write `if oldName in self.exp.routines`.